### PR TITLE
Updated Image with Correct URL

### PR DESCRIPTION
**Pull request by joinimran**
As required, I have updated the image with the correct URL. 

Problem: https://github.com/MicrosoftDocs/windows-itpro-docs/issues/8509

### CODE REVIEW

**Review by illfated — APPROVED** [review]
- [X] **Normal source view:** https://github.com/MicrosoftDocs/windows-itpro-docs/pull/8713/files
- [X] **Hide whitespace changes:** https://github.com/MicrosoftDocs/windows-itpro-docs/pull/8713/files?diff=unified&w=1
- [X] **Rich diff view:** https://github.com/MicrosoftDocs/windows-itpro-docs/pull/8713/files?diff=unified&w=1&short_path=1a84eed#diff-1a84eed248beed5b6f47cb49d6b993ca5e08a8042cd1babd8bab7d057702b8aa
- [X] **Image rich diff view:** https://github.com/MicrosoftDocs/windows-itpro-docs/pull/8713/files?short_path=d82adde#diff-d82addeadf260315de8dde6cd3b4d68258bc3ab9865e09ca461d779301e141dc

File: windows/security/identity-protection/hello-for-business/hello-feature-pin-reset.md [hunk 1]
```diff
@@ -66,26 +66,18 @@ You configure Windows 10 to use the Microsoft PIN Reset service using the comput
...
+2. Click **Endpoint Security**-> **Account Protection**-> **Properties**.
+3. Set **Enable PIN recovery** to **Yes**.
 
-    ```
-    dsregcmd /status | findstr -snip "tenantid"
-    ```
+> [!NOTE]
+> You can also setup PIN recovery using configuration profiles.
+> 1. Sign in to Endpoint Manager.
+> 2. Click **Devices** -> **Configuration Profiles** -> Create new profile or edit an existing profile using the Identity Protection profile type.
```
Comment (illfated) [inline windows/security/identity-protection/hello-for-business/hello-feature-pin-reset.md:78]: Minor cosmetic/readability suggestion:
```suggestion
> 2. Click **Devices** -> **Configuration Profiles** -> Create a new profile or edit an existing profile using the Identity Protection profile type.
```

**Comment by illfated** [comment]
Looks very good to me otherwise. Feel free to ask if there are any unanswered questions left.

**Comment by illfated** [comment]
Please note: whereas mapalko is the listed author for this document, he does not have merge access. Ask either the manager or anyone else working in this department when the PR has had its final review and handoff.

**Review by illfated — APPROVED** [review]
@e0i : For your information. At your service. As usual, mapalko is the current doc author, dansimp is the manager.

**Review by mapalko — APPROVED** [review]

**Comment by illfated** [comment]
#sign-off

**Pull request merged**